app/javascript/css/tailwind.scss

```diff
@@ -3,3 +3,32 @@
 @tailwind components;
 
 @tailwind utilities;
+
+.card {
+    @apply h-100 w-full flex items-center justify-center bg-teal-lightest font-sans ;
+}
+
+.innercard {
+    @apply bg-white rounded shadow p-6 m-4 w-full lg:max-w-lg ;
+}
+
+.cardtitle {
+    @apply mb-4 text-grey-darkest ;
+}
+
+.addfield {
+    @apply flex mb-4 ;
+}
+
+#listitem {
+    @apply flex mt-4 items-center ;
+}
+
+.addfieldplaceholder {
+    @apply shadow appearance-none border rounded w-full py-2 px-3 mr-4 text-grey-darker ;
+}
+
+.button {
+    @apply flex-no-shrink p-2 border-2 rounded text-teal border-teal hover:text-white hover:bg-teal ;
```

Comment:
The way you are writing this will break webpacker
```
.button {
    @apply flex-no-shrink p-2 border-2 rounded text-teal border-teal;
}
.button:hover {
    @apply text-white bg-teal ;
}
```
look over this 
https://tailwindcss.com/docs/functions-and-directives/#apply